test/clt-tests/sharding/test-dist-table-shards-5000.rec

```diff
@@ -0,0 +1,32 @@
+––– block: ../base/start-searchd-with-buddy –––
+––– input –––
+cat << 'EOF' > /tmp/create_tables.sh
+#!/usr/bin/env bash
```

Comment:
We can just run multiple command without writing to file and execute it, that should be simpler

```
locals=()
for n in $(seq 1 5000); do
	locals+=("local='system.t$n'")
	mysql -h0 -P9306 -e "create table system.t$n (id int)"
done
mysql -h0 -P9306 -e "create table t type='distributed' ${locals[*]}"
```